Bottom: 6291606da619cff024b4cc504f607b07afa39174
Top:    542b71b2abff523e6a8f46c9b30c981ef1c0b251
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-24 19:25:22 +0000

Refresh of polygon1-c-remove-second

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 567a473..a33584f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1686,7 +1686,7 @@ static void
 Collect (jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes,
 	 S_Rule s_rule, J_Rule j_rule)
 {
-  VNODE *cur, *other;
+  VNODE *cur; //, *other;
   DIRECTION dir;
 
   cur = &a->head;
@@ -1694,9 +1694,9 @@ Collect (jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes,
     {
       if (s_rule (cur, &dir) && cur->Flags.mark == 0)
 	Collect1 (e, cur, dir, contours, holes, j_rule);
-      other = cur;
-      if (other->cvc_prev)
-	Collect1 (e, other, dir, contours, holes, j_rule);
+//      other = cur;
+//      if (other->cvc_prev)
+//	Collect1 (e, other, dir, contours, holes, j_rule);
     }
   while ((cur = cur->next) != &a->head);
 }				/* Collect */
